src/site/content/en/blog/preload-scanner/index.md

```diff
@@ -236,16 +236,51 @@ That `rel=preload` hint is small, but it helps the browser discover the image so
+
+Take [this page](https://preload-scanner-fights.glitch.me/inline-nothing.html) as an example. In certain conditions the LCP candidate is the image at the top of the page, and the CSS is in a separate file loaded by a `<link>` element. The page also uses four web fonts which are requested as separate files from the CSS resource.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/GY0b2sh3wRt0CVpwdPvr.png", alt="A WebPageTest network waterfall chart of page with an external CSS file with four fonts referenced in it. The LCP candidate image is discovered by the preload scanner in due course.", width="800", height="297", loading="lazy" %}
+  <figcaption>
+    <strong>Fig. 12:</strong> A WebPageTest network waterfall chart of <a href="https://preload-scanner-fights.glitch.me/inline-nothing.html" rel="noopener">a web page</a> run on Chrome on a mobile device over a simulated 3G connection. The page's LCP candidate is an image loaded from an <code>&lt;img&gt;</code> element, but is discovered by the preload scanner because the CSS and the fonts required for the page load in separate resources, which avoids preventing the preload scanner from doing its job.
+  </figcaption>
+</figure>
+
+Now what happens if we inline the CSS _and_ inline all the fonts as base64 resources?
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ommDo1DKQUu1GxJ3YPvq.png", alt="A WebPageTest network waterfall chart of page with an external CSS file with four fonts referenced in it. The preload scanner is delayed significantly from discovering the LCP image .", width="800", height="297", loading="lazy" %}
+  <figcaption>
+    <strong>Fig. 13:</strong> A WebPageTest network waterfall chart of <a href="https://preload-scanner-fights.glitch.me/inline-everything.html" rel="noopener">a web page</a> run on Chrome on a mobile device over a simulated 3G connection. The page's LCP candidate is an image loaded from an <code>&lt;img&gt;</code> element, but the inlining of the CSS and its four font resources delays the preload scanner from discovering the image until those resources are fully parsed.
+  </figcaption>
+</figure>
+
+The impact of inlining yields negative consequences for LCP in this example&mdash;and for performance in general. The version of the page that doesn't inling anything paints the LCP image in about 3.5 seconds. The page that inlines everything doesn't paint the LCP image until roughly 6.4 seconds.
+
+To be candid, there's more at play here than just the preload scanner. Inlining fonts is not a great strategy in part because external font resources aren't downloaded unless they're determined necessary by the CSSOM. When those fonts are inlined as base64, however, they're downloaded whether they're needed or not.
```

Comment:
```suggestion
To be candid, there's more at play here than just the preload scanner. Inlining fonts is not a great strategy because Base64 is an inefficient format for binary resources. Additionally, because external font resources aren't downloaded unless they're determined necessary by the CSSOM but when those fonts are inlined as Base64, they're downloaded whether they're needed or not.
```